subgraphs/_reference_/src/common/utils/numbers.ts

```diff
@@ -39,3 +39,15 @@ export function calculateMedian(prices: BigDecimal[]): BigDecimal {
 
   return sorted[mid - 1];
 }
+
+// Ray is 27 decimal Wad is 18 decimal
+
+export function rayToWad(a: BigInt): BigInt {
```

Comment:
Maybe add a link to Aave's (or Maker's) definition or ray/wad as it's not the most commonly used.